shared/src/types/address.rs

```diff
@@ -43,6 +43,10 @@ mod internal {
         "ano::Inter-Blockchain Communication          ";
     pub const PARAMETERS: &str =
         "ano::Protocol Parameters                     ";
+    pub const BURN: &str =
+        "ano::Burn Address                            ";
+    pub const MINT: &str =
+        "ano::Mint Address                            ";
```

Comment:
let's prefix these: e.g.:
```suggestion
    pub const IBC_BURN: &str =
        "ano::IBC Burn Address                        ";
    pub const IBC_MINT: &str =
        "ano::IBC Mint Address                        ";
```